src/example-types/facet.js

```diff
@@ -134,5 +150,64 @@ module.exports = {
           }),
         options
       ),
-    })),
+    }))
+
+    let lostIds = _.difference(
+      valueIds,
+      _.map(
+        ({ name }) => F.when(node.isMongoId, _.toString, name),
+        results.options
+      )
+    )
+
+    let maybeMapObjectId = missedIds =>
+      node.isMongoId ? _.map(ObjectId, missedIds) : missedIds
```

Comment:
Have to use if else in this way, Cause if there is no isMongoId attribute in node.isMongoId is undefined. we have condition undefined bug when we use F.when